src/table/Root.tsx

```diff
@@ -102,7 +103,7 @@ export function renderVirtualizedTable(props: VirtualTableRenderProps, reactRoot
           pageInfo={pageInfo}
           initialDataPages={initialDataPages}
         >
-          <VirtualizedTable />
+          <VirtualizedTable viewService={viewService} />
```

Comment:
Not sure if we want to put it here or on the context. Especially since there are no other props passed to `VirtualizedTable`

Reply:
I updated it by moving viewService to the context.